src/main/services/mods/bs-mods-manager.service.ts

```diff
@@ -207,13 +206,11 @@ export class BsModsManagerService {
             processIPA.once("exit", code => {
                 clearTimeout(timeout);
                 if (code === 0) {
-                    log.info("Ipa process exist with code 0");
```

Comment:
For the record, I don't think this is ever necessary as it would time out, but feel free to change this if I'm wrong.